homeassistant/components/danfoss_air/sensor.py

```diff
@@ -27,67 +27,89 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             TEMP_CELSIUS,
             ReadCommand.exhaustTemperature,
             DEVICE_CLASS_TEMPERATURE,
+            STATE_CLASS_MEASUREMENT,
```

Comment:
Not for now, but for a future PR this should be upgraded to use `SensorEntityDescription`s instead.

Reply:
Will see if I get time to look at that in the near future.